### ISSUE TITLE
Complete AJAX in the MCP

### ISSUE THREAD

**Issue by iMattPro** [phpbb-extensions/cannedmessages#8]
Only moving uses AJAX. Other functions like Deleting could use AJAX too.

Also should fix non ajax interactions, for example, when deleting but then aborting the confirm box by clicking no, the resulting page is an empty list, not a proper list or the list you were viewing previously.

Also some of the URL's look a bit weird. After deleting (clicking no) there's leftover parameters that shouldn't be there.

Basically the MCP controller needs a bit of retooling.
